website/layouts/shortcodes/script.html

```diff
@@ -1,3 +1,5 @@
-{{ $file := .Get "file" | readFile }}
+{{ $file := .Get "file" }}
```

Comment:
Cool! Can we call this file version.html?

Reply:
The name of the files corresponds to the shortcode tag, e.g. `{{% script file="...` so if we change the name of the file we will have to use `{{% version` which would be a bit confusing.

Reply:
ahhh gotcha.